packages/extensions/modelerfour/src/modeler/constants.ts

```diff
@@ -0,0 +1 @@
+export const KnownSpecialHeaders = ["Repeatability-Request-Id", "Repeatability-First-Send"];
```

Comment:
Should we add Accept, Content-Type, and those headers that we were considering different already?

Reply:
So those are technically not allowed by openapi to start with. However when defining them it was because one wanted to be able to specify the content-type manually. I think for storage batch request it needed that.

I originally wanted to have the default `ignore-headers` be using this but it doesn't look like we have defaults here and I don't see any other generator using it right now.

If there are some that are already manually ignored then I can look at it.

Reply:
x-ms-request-client-id? Do you have anything about this one already somewhere?

Reply:
can't see being used in core, python and typescript generator.